mysql-test/suite/ndb/t/bug_rondb-852.test

```diff
@@ -0,0 +1,33 @@
+# RONDB-852 bugfix
+-- source include/have_ndb.inc
+SHOW VARIABLES LIKE 'log_bin';
+SHOW VARIABLES LIKE 'ndb_log_bin';
+
+CREATE LOGFILE GROUP lg_1
+    ADD UNDOFILE 'undo_1.log'
+    INITIAL_SIZE 16M
+    UNDO_BUFFER_SIZE 2M
+    ENGINE NDBCLUSTER;
+
+CREATE TABLESPACE ts_1
+    ADD DATAFILE 'data_1.dat'
+    USE LOGFILE GROUP lg_1
+    INITIAL_SIZE 32M
```

Comment:
To ensure that test case works I think you need to add one more line:
EXTENT_SIZE 32k
to the CREATE TABLESPACE